salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -106,6 +93,21 @@ server {
 
     location / {
         try_files $uri /app.php$is_args$query_string;
+
+        add_header Cache-Control "{{ pillar.journal.cache_control }}";
```

Comment:
this has a good default but these redirect should maybe warrant a very long expiration time? They are changed only if there's a bug so hours could possibly work?

Reply:
These are for legacy paths, and since we can clear easily let's go for a week.